lib/reek/rake/task.rb

```diff
@@ -32,118 +33,87 @@ module Rake
     #   rake reek REEK_OPTS=-s                   # sorts the report by smell
     #
     class Task < ::Rake::TaskLib
-      # Name of reek task.
-      # Defaults to :reek.
-      attr_accessor :name
+      # Name of reek task. Defaults to :reek.
+      attr_writer :name
```

Comment:
I basically changed all attr_accessors to attr_writers because that is all we need.